domains/polo.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "polo-1245-oficial",
+           "email": "81937184+polo-1245-oficial@users.noreply.github.com",
+           "discord": "725316907187568701"
+        },
+    
+        "record": {
+            "CNAME": "polo-1245-oficial.github.io"
```

Comment:
Page seems to give a 404 error, if you have the site hosted then please send a preview of it

Reply:
preview <img width="1080" alt="image" src="https://github.com/is-a-dev/register/assets/81937184/08056de7-c0ea-4db1-b0e5-4bf58f92a383">